opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/JobManager.java

```diff
@@ -613,6 +614,93 @@ public OpenCGAResult<Job> submit(String studyStr, String toolId, Enums.Priority
         }
     }
 
+    public OpenCGAResult<Job> rescheduleJobs(String studyStr, List<Long> jobUids, String scheduledStartTime, String token)
+            throws CatalogException {
+        return rescheduleJobs(studyStr, jobUids, scheduledStartTime, null, token);
+    }
+
+    public OpenCGAResult<Job> rescheduleJobs(String studyStr, List<Long> jobUids, String scheduledStartTime, String eventMessage,
+                                             String token) throws CatalogException {
+        JwtPayload tokenPayload = catalogManager.getUserManager().validateToken(token);
+        CatalogFqn studyFqn = CatalogFqn.extractFqnFromStudy(studyStr, tokenPayload);
+        String organizationId = studyFqn.getOrganizationId();
+        String userId = tokenPayload.getUserId(organizationId);
+
+        ObjectMap auditParams = new ObjectMap()
+                .append("study", studyStr)
+                .append("jobUids", jobUids)
+                .append("scheduledStartTime", scheduledStartTime)
+                .append("eventMessage", eventMessage)
+                .append("token", token);
+        try {
+            Study study = catalogManager.getStudyManager().resolveId(studyFqn, tokenPayload);
+            authorizationManager.checkIsAtLeastStudyAdministrator(organizationId, study.getUid(), userId);
```

Comment:
I understand that this is a "delicate" operation that should only be executed from the execution daemon (to avoid race conditions between the rest and the master)
Should it check for opencga user instead? And add some javadocs informing about the scope of the method?